plugins/org.yakindu.sct.generator.genmodel/src/org/yakindu/sct/generator/genmodel/validation/SGenJavaValidator.java

```diff
@@ -148,6 +149,14 @@ public void checkGeneratorExists(GeneratorModel model) {
 					SGenPackage.Literals.GENERATOR_MODEL__GENERATOR_ID);
 		}
 	}
+	
+	@Check
+	public void checkEntriesExist(GeneratorModel model) {
+		if(model.getEntries() == null || model.getEntries().isEmpty()) {
```

Comment:
model.getEntries is never null, EMF one to many references always return empty lists so the == null check is obsolete